internal/controller/api/v1alpha1/agent_types.go

```diff
@@ -130,7 +130,6 @@ type AgentSpec struct {
 	Role                    models.HostRole   `json:"role" protobuf:"bytes,1,opt,name=role,casttype=HostRole"`
 	Hostname                string            `json:"hostname,omitempty"`
 	MachineConfigPool       string            `json:"machineConfigPool,omitempty"`
-	Enabled                 *bool             `json:"enabled,omitempty"`
 	IgnitionConfigOverrides string            `json:"ignitionConfigOverrides,omitempty"`
```

Comment:
need to remove `IgnitionConfigOverrides and `InstallerArgs` as well

Reply:
Done